### PR TITLE
chore(Javascript): add fibonacci series

### PR DESCRIPTION
**Pull request by ValerioCipolla**
**PR Checklist:**
- [x] My submission is formatted according to the guidelines in the [contributing guide](https://github.com/MakeContributions/DSA#contribution-guidelines)
- [x] My addition is on refer on the language README.md file
- [x] My addition does not have a spelling problem
- [x] My submission has a proper and user-friendly description of the algorithm
- [x] My submission has the time complexity of the algorithm
- [x] My submission has sample input-output of the program (NOT FOR PYTHON)

**What kind of change does this PR introduce?** (check at least one)

- [ ] Bugfix
- [x] New algorithm
- [ ] Optimization in previous algorithms
- [ ] Code style update
- [ ] Refactor
- [x] Documentation
- [ ] Other, please describe:

**Briefly describe the changes in this PR**

**Other information:**

It's my first contribution to the project, hopefully I did everything right. I created a new folder in the javascript folder called "maths" for all math-related algos, and created a first one for the fibonacci series. If did anything wrong and needs changing please let me know and I will try and fix it asap.

Thank you

### CODE REVIEW

File: algorithms/JavaScript/src/maths/fibonacci-series.js [hunk 1]
```diff
@@ -0,0 +1,23 @@
+// algorithm to generate first n numbers of fibonacci series
+
+function fibonacci(n) {
+  // Initialize array 'fibS' with the first two numbers of the fibonacci series
+  // [0, 1]
+  const fibS = [0, 1];
```
Comment (ming-tsai) [inline algorithms/JavaScript/src/maths/fibonacci-series.js:6]: Rename the parameter to `series`
```suggestion
  const series = [0, 1];
```

**Review by ming-tsai — CHANGES_REQUESTED** [review]
Hi @ValerioCipolla,
Please solved the code spelling problem
image

File: algorithms/JavaScript/src/maths/fibonacci-series.js (hunk 1 above)
Comment (ValerioCipolla) [reply algorithms/JavaScript/src/maths/fibonacci-series.js]: done ✔️  
Please could you explain to me why that was needed? 
Thank you

**Comment by ValerioCipolla** [comment]
Hi @ming-tsai  sorry about that, I didn't understand what the issue was. hopefully that's fixed now, also, I added the fibonacci series files to the required ones in index.js, is that the right thing to do? Sorry I am just starting out now with contributing to open source projects. Thank you for being patient.

**Comment by ming-tsai** [comment]
Hi @ValerioCipolla,
Sorry, I forgot to tell you to add the time complexity of the algorithm on the `.js` file.

**Comment by ValerioCipolla** [comment]
done ✔️ Let me know if you need more changes before you can merge

**Review by ming-tsai — APPROVED** [review]

**Pull request merged**